Index: src/main/java/com/module/controller/MiddleController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/module/controller/MiddleController.java b/src/main/java/com/module/controller/MiddleController.java
--- a/src/main/java/com/module/controller/MiddleController.java	(date 1652589616093)
+++ b/src/main/java/com/module/controller/MiddleController.java	(date 1652589616093)
@@ -41,7 +41,24 @@
         model.addAttribute("courseId", id);
         return "manage/middle/middleList";
     }
-4
+
+    /**
+     * 跳转到课程目标达成度页面
+     *
+     * @return
+     */
+    @RequestMapping("manage/achiveInfo")
+    public String achiveInfo(Model model,Integer courseId) {
+        model.addAttribute("courseId", courseId);
+        List<Big> bigs = bigMapper.selectAll(null);
+        for (Big big : bigs) {
+            big.setSmalls(smallMapper.selectByBigId(big.getId()));
+        }
+        model.addAttribute("bigs", bigs);
+        return "manage/middle/saveMiddle";
+    }
+
+
     @Autowired
     private BigMapper bigMapper;
 
